### PR TITLE
update sslProtocol attribute description

### PR DESCRIPTION
**Pull request by acdemyers**
Updating sslProtocol description to account for multiple protocol support

### CODE REVIEW

**Comment by acdemyers** [comment]
#build

File: dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -30,7 +30,7 @@ keystore.desc=A keystore containing key entries for the SSL repertoire. This att
 truststore=Truststore
 truststore.desc=A keystore containing trusted certificate entries used by the SSL repertoire for signing verification. This attribute is optional. If unspecified, the same keystore is used for both key and trusted certificate entries.
 repertoire.sslProtocol=SSL protocol
-repertoire.sslProtocol.desc=The SSL handshake protocol. Protocol values can be found in the documentation for the underlying JRE's Java Secure Socket Extension (JSSE) provider.  When using the IBM JRE the default value is SSL_TLSv2 and when using the Oracle JRE the default value is SSL.
+repertoire.sslProtocol.desc=The SSL handshake protocol. Protocol can be a single value found in the documentation for the underlying JRE's Java Secure Socket Extension (JSSE) provider.  When using the IBM JRE the default value is SSL_TLSv2 and when using the Oracle JRE the default value is SSL.  The protocol can also be a comma separated list of protocol values from TLSv1, TLSv1.1, TLSv1.2, TLSv1.3 values.
```
Comment (jantley-ibm) [inline dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties]: Change:
Protocol can be
to:
The protocol can be set to

Change:
The protocol can also be a comma separated list of protocol values from TLSv1, TLSv1.1, TLSv1.2, TLSv1.3 values.
to
The protocol can also be a comma-separated list of any of the following values: TLSv1, TLSv1.1, TLSv1.2, or TLSv1.3.

**Review by jantley-ibm — COMMENTED** [review]
Suggested ID edits, let me know if you have questions. I will add the "ID reviewed" tag when we agree on changes. Thanks!

**Review by emilytee — APPROVED** [review]

File: dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (acdemyers) [reply dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties]: @jantley-ibm  The message has been updated please review.

**Comment by acdemyers** [comment]
#build

File: dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (jantley-ibm) [reply dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties]: @acdemyers I see the second change but not the first, was that intentional?

Comment (acdemyers) [reply dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties]: @jantley-ibm Sorry my error.  I missed putting that.  It's been updated please review again.

**Comment by acdemyers** [comment]
#libbybot

File: dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (jantley-ibm) [reply dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties]: @acdemyers looks good, thanks! I added the ID reviewed tag.

**Comment by acdemyers** [comment]
#libbybot

**Pull request merged**